pymysql/connections.py

```diff
@@ -374,6 +396,9 @@ def is_auth_switch_request(self):
         # http://dev.mysql.com/doc/internals/en/connection-phase-packets.html#packet-Protocol::AuthSwitchRequest
         return self._data[0:1] == b'\xfe'
 
+    def is_rsa_public_key(self):
+        return self._data[0:1] == b'\x01'
```

Comment:
Is `\x01` really RSA_PUBLIC_KEY packet?
Isn't it a more general purpose packet which has other semantics in other context?